### PR TITLE
Fix broken tests

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
Fixed the TravisCI build

> 

Not animooted. -1

**Comment by thiemowmde** [comment]
What's wrong with Scrutinizer?

**Comment by JeroenDeDauw** [comment]
It failed randomly. I amended the commit message and now it passes fine.

File: tests/unit/Serializers/ClaimSerializerTest.php [hunk 1]
```diff
@@ -190,7 +190,7 @@ public function serializationProvider() {
 
 		$claim = new Statement( new Claim( new PropertyNoValueSnak( 42 ) ) );
 		$claim->setReferences( new ReferenceList( array(
-			new Reference()
+			new Reference( array( new PropertyNoValueSnak( 1 ) ) )
```
Comment (thiemowmde) [inline tests/unit/Serializers/ClaimSerializerTest.php]: This test case does not make sense any more. The `references > snaks` array below is empty. How is that possible?

File: tests/unit/Serializers/ReferenceListSerializerTest.php [hunk 2]
```diff
@@ -65,10 +65,15 @@ public function serializationProvider() {
 					array(
 						'hash' => 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
 						'snaks' => array()
+					),
+					array(
+						'hash' => 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
+						'snaks' => array()
```
Comment (thiemowmde) [inline tests/unit/Serializers/ReferenceListSerializerTest.php]: Same as above. How is it possible that a Reference with 1 snak is serialized as an empty array?

**Comment by thiemowmde** [comment]
-1, see the comments above.

File: tests/unit/Serializers/ClaimSerializerTest.php (hunk 1 above)
Comment (JeroenDeDauw) [inline tests/unit/Serializers/ClaimSerializerTest.php]: This is a unit test for the ClaimSerializer, so it does not use the ReferenceSerializer. Instead, a simple fake is used.

**Pull request merged**